services/cron-service/env-defaults

```diff
@@ -12,3 +12,9 @@ WORK_REAPER_BATCH_SIZE=2000
 
 # The cron schedule for restarting Prometheus if it is down
 RESTART_PROMETHEUS_CRON="*/10 * * * *" # every 10 minutes
+
+# The cron schedule for the user-work updater
+USER_WORK_UPDATER_CRON="*/30 * * * *" # every thirty minutes
+
+# Rows in user-work older than this will cause the user-work for a job to be recalculated
+USER_WORK_EXPIRATION_MINUTES="60"
```

Comment:
Any reason this one needs to be a string (for other parameters like this we use integers e.g. REAPABLE_WORK_AGE_MINUTES=20160)?